wallet/res/layout/fragment_enter_pin.xml

```diff
@@ -57,13 +57,13 @@
                         android:layout_marginTop="8dp">
 
                         <LinearLayout
-                            android:layout_width="wrap_content"
+                            android:layout_width="match_parent"
```

Comment:
The goal here is to widen the area where warning text about remaining PIN attempts can be displayed.

“Wrong PIN! 2 attempts” was displayed instead of “Wrong PIN! 2 attempts remaining” - other languages may have more than one word clipped off from the warning text.